.github/workflows/ci.yaml

```diff
@@ -141,12 +143,11 @@ jobs:
         # Triggered only on tag creation
         if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
         run: |
-          sudo apt install zip -y
           ls build
           mv build/main.wasm build/coraza-proxy-wasm.wasm
           tag="${GITHUB_REF#refs/tags/}"
           zip -j build/coraza-proxy-wasm-${tag}.zip build/coraza-proxy-wasm.wasm
-          ./.github/workflows/release_notes.sh ${tag} > release-notes.txt
+          sudo ./.github/workflows/release_notes.sh ${tag} > release-notes.txt
```

Comment:
Do we reallyh need sudo?

Reply:
Trying to fix `./.github/workflows/release_notes.sh: Permission denied`. I might add a line with `chmod +x`, it could do the trick.

Reply:
Did you chmod in local?

Reply:
❤️🙇🏻‍♂️